AnnoyancesFilter/sections/self-promo.txt

```diff
@@ -911,6 +911,7 @@ subbota.ua##.textwidget > div#slidebox
 justnutritive.com##.top-promotion
 forum.doctorhead.ru###content > div[id="secondary_navigation"] ~ noscript + center
 apteka.ru##.apteka-adv
+stocksregister.com##a[class^="manual_trigger"]
```

Comment:
Something like that `stocksregister.com##.td-post-content > h3[style] a[class^="manual_trigger"]` should be ok.

Reply:
What's the problem here: `stocksregister.com##a[class^="manual_trigger"]`. `manual_trigger` appears only once, so I do not think specification is needed. Narrowed enough.

Reply:
This rule is more accurate and it shouldn't cause false positives.


> appears only once

Not really safe.

Reply:
Okay.